spec/requests/claims_spec.rb

```diff
@@ -165,6 +165,12 @@
       end
 
       context "when updating from check-your-answers" do
+        let(:claim_mailer_spy) { spy(:mailer) }
+        before do
+          allow(ClaimMailer).to receive(:submitted).with(in_progress_claim)
```

Comment:
Rather than stubbing the mailer I think it's better to check the contents of `ActionMailer::Base.deliveries` as part of the expectation: tests the expected outcome, not the implementation.